packages/okta-react/README.md

```diff
@@ -8,30 +8,19 @@ Okta React SDK makes it easy to integrate [react-router](https://github.com/Reac
 This library currently supports:
   - [OAuth 2.0 Implicit Flow](https://tools.ietf.org/html/rfc6749#section-1.3.2)
 
-## Prerequisites
-* If you do not already have a **Developer Edition Account**, you can create one at [https://developer.okta.com/signup/](https://developer.okta.com/signup/).
-* If you don't have a React app, or are new to React, please continue with the [React Quickstart](https://facebook.github.io/react/docs/installation.html#creating-a-new-application) guide. It will walk you through the creation of a React app, creating routes, and other application development essentials.
-
-## Add an OpenID Connect Client in Okta
```

Comment:
This information is already present in the “Helpful Links” section. Removing it hence.